tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot

```diff
@@ -1,23 +1,149 @@
 *** Settings ***
-Resource        ../../../Resources/Page/LoginPage.robot
-Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
 Library         SeleniumLibrary
+Library         OpenShiftCLI
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
+Resource        ../../../Resources/Page/HybridCloudConsole/Rhosak.robot
+Resource        ../../../Resources/Page/ODH/JupyterHub/LoginJupyterHub.robot
+Resource        ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource        ../../../Resources/Page/ODH/JupyterHub/JupyterLabSidebar.robot
+Resource        ../../../Resources/Page/ODH/AiApps/Rhosak.robot
 Suite Setup     Kafka Suite Setup
```

Comment:
Just a suggestion ..I think it would be better not to call individual resource file(.robot)  instead we can call the one common resource file which has all these file already present(.resource). For example : We can remove all the Jupyterhub robot file from above and just call ODHJupyterhub.resource which has all those file imported there. What do you think

Reply:
I think this is a great suggestion and something that I never got around to doing, +1 from me

Reply:
agree with this, thanks!